software_attributes/schema.py

```diff
@@ -171,10 +171,44 @@
                 }
             ],
         },
+        "turbine_model": {
+            "description": "The fidelity with which the turbine is represented in the software model (if at all).",
+            "type": "array",
+            "items": {
+                "enum": ["regression",
+                         "lookup_table",
+                         "momentum_sink",
+                         "multibody_bem",
+                         "vortex_method",
+                         "actuator",
+                         "geometry_resolved",
+                         "SCADA",
+                         "N/A"]
+            }
+        },
+        "wake_model": {
+            "description": "The fidelity with which turbine wakes are represented in the software model (if at all). ",
+            "type": "array",
+            "items": {
+                "enum": ["analytical",
+                         "dynamic_meandering",
+                         "vortex_method",
+                         "navier_stokes",
+                         "N/A"]
+            }
+        },
         "loads": {
             "description": "Does the software calculate system loads?",
             "type": "boolean"
         },
+        "design_optimization": {
```

Comment:
This is also included in the "primary_uses" field. Do you think it would be more than a boolean eventually? If so, maybe we leave it here and remove it above. Maybe the `design_optimization` field could list some types of optimization techniques and design methods.

Reply:
I missed that primary uses field.  Might be worth a conversation on whether there should be booleans like this one or the primary uses.  For instance, on FLORIS it can do "controls", "design", and "optimization".  Which one is the primary use?  Would it be more clear to have those in these boolean fields (with the downside of adding many more columns in the comparison tables)?